.github/workflows/argus-docker-build.yaml

```diff
@@ -52,14 +52,21 @@ jobs:
     permissions:
       id-token: write
       contents: read
+      pull-requests: read
     steps:
-      - uses: chanzuckerberg/github-actions/.github/actions/argus-builder/build-prep@c210eb8dba1aeb3ccf02186ad89c3c85f6fc4da7
+      - uses: 8BitJonny/gh-get-current-pr@3.0.0
+        id: findPr
+        with:
+          filterOutClosed: true
+          filterOutDraft: false
+      - uses: chanzuckerberg/github-actions/.github/actions/argus-builder/build-prep@rzheng/stack-label
```

Comment:
what is our versioning strategy? continue using a sha1? leave it pointing to a branch like this PR? maybe we can create a tag?

Reply:
we use `release-please` to propose releases based on our conventional commits and merging the release proposal PR creates the actual release/tag. the reason this uses a sha instead of one of those tags is that it allows us to make a single release with the update instead of first releasing `build-prep` and then coming back and updating the version of `build-prep` used in this file and making another release